en/integrations/plugin-packs/procedures/network-cisco-firepower-snmp.md

```diff
@@ -0,0 +1,181 @@
+| interface status                                         | Status of the interface                                 |      |
+| *interfacename*#interface.traffic.in.bitspersecond       | Incoming traffic going through the interface.           | b/s  |
+| *interfacename*#interface.traffic.out.bitspersecond      | Outgoing traffic going through the interface.           | b/s  |
+| *interfacename*#interface.packets.in.error.percentage    | Incoming errored packets going through the interface.   | %    |
+| *interfacename*#interface.packets.in.discard.percentage  | Incoming discarded packets going through the interface. | %    |
+| *interfacename*#interface.packets.out.error.percentage   | Outgoing errored packets going through the interface.   | %    |
+| *interfacename*#interface.packets.out.discard.percentage | Outgoing discarded packets going through the interface. | %    |
+
+It is possible to filter on the name of an interface using a REGEXP of the form [```--interface='^eth1/0/1$' --name```].
+
+<!--Hardware-->
+
+| Metric name                                    | Description                     | Unit |
+| :--------------------------------------------- | :------------------------------ | :--- |
+| chassis status                                 | Status of the chassis           |      |
+| *dn*#hardware.chassis.input.power.watt         | Input power of the chassis      | W    |
+| *dn*#hardware.chassis.output.power.watt        | Output power of the chassis     | W    |
+| cpuunit status                                 | Status of the cpu unit          |      |
+| *dn*#hardware.cpuunit.temperature.celsius      | Status of the cpu unit          | C    |
+| fan status                                     | Status of the fan               |      |
+| *dn*#hardware.fan.speed.rpm                    | Speed of the fan                | rpm  |
+| fanmodule status                               | Status of the fan module        |      |
+| *dn*#hardware.fanmodule.temperature.celsius    | Temperature of the fan module   | C    |
+| memoryunit status                              | Status of the memory unit       |      |
+| *dn*#hardware.memoryunit.temperature.celsius   | Temperature of the memory unit  | C    |
+| psu status                                     | Status of the power supply      |      |
+| *dn*#hardware.powersupply.temperature.celsius  | Temperature of the power supply | C    |
+
+<!--Memory-->
+
+| Metric name                              | Description                | Unit |
+| :--------------------------------------- | :------------------------- | :--- |
+| *securitymodule*#memory.usage.bytes      | Memory usage               | B    |
+| *securitymodule*#memory.free.bytes       | Free memory                | B    |
+| *securitymodule*#memory.usage.percentage | Memory usage in percentage | %    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To control your Cisco Firepower, the SNMP must be configured.
```

Comment:
Any link to guide user to the official documentation. 

E.g: https://www.cisco.com/c/en/us/support/docs/ip/simple-network-management-protocol-snmp/213971-configure-snmp-on-firepower-ngfw-applian.html